<launch>
  <arg name="robot_ip" default="192.168.3.108"/>
  <include file="$(find franka_control)/launch/franka_control.launch" pass_all_args="true"/>
  <arg name="arm_id" default="panda"/>
  <rosparam command="load" file="$(find franka_controllers)/config/controller_parameters.yaml" subst_value="true" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="LCQP_controller_frankaModel"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_controllers)/launch/franka_with_marker.rviz -f $(arg arm_id)_link0 - -splash-screen $(find franka_visualization)/splash.png"/>
  <node name="interactive_marker" pkg="franka_controllers" type="interactive_marker.py" required="true" output="screen">
    <param name="link_name" value="$(arg arm_id)_link0" />
    <remap from="obstacle_pose" to="/LCQP_controller_frankaModel/obstacle_pose" />
  </node>
</launch>
